pkg/cmd/registry/artifact/util/constants.go

```diff
@@ -19,7 +19,18 @@ var AllowedArtifactTypeEnumValues = []string{
 	"XML",
 }
 
+var AllowedArtifactStateEnumValues = []string{
+	"ENABLED",
+	"DISABLED",
+	"DEPRECATED",
+}
```

Comment:
> Technically not a "const" 😉

Arrays cannot be constants. 

> It might be good to also create constants for these values to make them referencable in a switch statement or if etc.

Those values are actually in SDK but we cannot reference them because they are private: 
We can:

- Have them as part of the SDK (possibly much better idea)
- Use custom templates and make those public
- Wait for issue to be seen and fixed :D
https://github.com/OpenAPITools/openapi-generator/issues/10385

Reply:
> Arrays cannot be constants.

yeah I know :)

Reply:
> Use custom templates and make those public

I was thinking about this lately. Custom templates may still be the better option as an alternative is to write SDKs from scratch. Both require maintenance but the former has much more maintenance.